.gitlab-ci.yml

```diff
@@ -255,6 +282,57 @@ trigger_devops:
     - if: $CI_COMMIT_BRANCH == "main"
   trigger: lg/identity-devops
 
+review-app:
+  stage: review
+  needs:
+    - job: build-idp-image
+  resource_group: $CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov
+  image:
+    name: dtzar/helm-kubectl:latest
+  script:
+    - kubectl config get-contexts
+    - export CONTEXT=$(kubectl config get-contexts | grep -v CURRENT | awk '{print $1}' | head -1)
+    - kubectl config use-context "$CONTEXT"
+    - >-
+      helm upgrade --install --namespace review-apps
+      --debug
+      --set idp.image.repository="${ECR_REGISTRY}/identity-idp/review"
+      --set idp.image.tag="${CI_COMMIT_SHA}"
+      --set idp.ingress.enabled=true
+      --set-json idp.env="[{\"name\":\"POSTGRES_SSLMODE\",\"value\":\"prefer\"},{\"name\":\"POSTGRES_NAME\",\"value\":\"idp\"},{\"name\":\"POSTGRES_HOST\",\"value\":\"$CI_ENVIRONMENT_SLUG-identity-idp-chart-postgres.review-apps\"},{\"name\":\"POSTGRES_USERNAME\",\"value\":\"postgres\"},{\"name\":\"POSTGRES_PASSWORD\",\"value\":\"postgres\"},{\"name\":\"POSTGRES_WORKER_SSLMODE\",\"value\":\"prefer\"},{\"name\":\"POSTGRES_WORKER_NAME\",\"value\":\"idp-worker-jobs\"},{\"name\":\"POSTGRES_WORKER_HOST\",\"value\":\"$CI_ENVIRONMENT_SLUG-identity-idp-chart-postgres.review-apps\"},{\"name\":\"POSTGRES_WORKER_USERNAME\",\"value\":\"postgres\"},{\"name\":\"POSTGRES_WORKER_PASSWORD\",\"value\":\"postgres\"},{\"name\":\"LOGIN_ENV\",\"value\":\"dev\"},{\"name\":\"RAILS_OFFLINE\",\"value\":\"true\"},{\"name\":\"REDIS_IRS_ATTEMPTS_API_URL\",\"value\":\"redis://$CI_ENVIRONMENT_SLUG-identity-idp-chart-redis.review-apps:6379/2\"},{\"name\":\"REDIS_THROTTLE_URL\",\"value\":\"redis://$CI_ENVIRONMENT_SLUG-identity-idp-chart-redis.review-apps:6379/1\"},{\"name\":\"REDIS_URL\",\"value\":\"redis://$CI_ENVIRONMENT_SLUG-identity-idp-chart-redis.review-apps:6379\"},{\"name\":\"ASSET_HOST\",\"value\":\"https://$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov\"},{\"name\":\"DOMAIN_NAME\",\"value\":\"$CI_ENVIRONMENT_SLUG.review-app.identitysandbox.gov\"}]"
```

Comment:
is there a way we could use heredoc syntax or something to pretty format it to make editing this easier? or put it in a file and read it?

I'm imaginging like
```
idp_env=$(cat <<EOS
[
  {"name": "POSTGRES_SSLMODE", "value": "prefer"},
]
EOS
)

Reply:
Good suggestion. Done.